README.md

```diff
@@ -46,22 +46,21 @@ Note: the user agent lists differ there.
 
 ### Render
 ```
-/render/<url>
+GET /render/<url>
 ```
 
-The `render` endpoint will render your page and serialize your page. Available options:
- * `wc-inject-shadydom` default `false` - used to correctly render Web Components v1. See
- [Using with web components](#web-components) for more information.
+The `render` endpoint will render your page and serialize your page.
 
 ### Screenshot
 ```
-/screenshot/<url>
+GET /screenshot/<url>
+POST /screenshot/<url>
 ```
 
-The `screenshot` endpoint can be used to verify that your page is rendering correctly.
-Available options:
- * `width` default `1000` - used to set the viewport width (max 2000)
- * `height` default `1000` - used to set the viewport height (max 2000)
+The `screenshot` endpoint can be used to verify that your page is rendering
+correctly. Options are available as a JSON string in the `POST` body (options for `GET` are unsupported). See
+[Puppeteer documentation](https://github.com/GoogleChrome/puppeteer/blob/v1.6.0/docs/api.md#pagescreenshotoptions)
+for available options.
```

Comment:
Add that you can't modify type or encoding.

Reply:
Done.